### DISCUSSION TITLE
现实已经有收入奇亚比，而自己钱包里面没有

### DISCUSSION THREAD

**Discussion by longteng023** [Chia-Network/chia-blockchain#5098] [Support]
现实自己得到奇亚币，而自己的钱包 还是早前的2枚， 后面的5月8号1.63和5月9号0.75这里的2个币没有，
<HTML><BODY>

收入 | xch1lszr8uet43um7hrz4c9luqa9sejyxplqm3g9g789nyt5ymrc8hlsnm098y | 05/13/2021 下午11:17:39 | 区块高度确认 274544 | 0.0000000001 | 0
-- | -- | -- | -- | -- | --
收入 | xch1lszr8uet43um7hrz4c9luqa9sejyxplqm3g9g789nyt5ymrc8hlsnm098y | 05/09/2021 上午8:55:11 | 区块高度确认 252811 | 0.75 | 0
收入 | xch1nd86002lt5f95afe2vwpr9q0h50parm4uttwsm84n2j5p65rfzfs6w69xq | 05/08/2021 上午8:55:49 | 区块高度确认 247748 | 1.63 | 0
收入 | xch1x74usf9upy679qaf3m6kxgf23e0ks9tt6x2hnmxr0p7tjw8s5mkq4derw3 | 04/29/2021 上午6:29:51 | 区块高度确认 181925 | 1.75 | 0
收入 | xch1x74usf9upy679qaf3m6kxgf23e0ks9tt6x2hnmxr0p7tjw8s5mkq4derw3 | 04/29/2021 上午6:29:51 | 区块高度确认 181925 | 0.25 | 0

</BODY></HTML>

**Comment by langlangaaa** [comment]
我的问题跟你一样..显示爆块  钱包里面也没有

**Comment by longteng023** [comment]
不知道为什么 你找到原因没有

**Comment by cccat6** [comment]
在Chia Explorer上查查 www.chiaexplorer.com
如果这上面没有就是真没有。这上面看到的是最终全网同步的区块。有可能是你自己在小范围里确认成功了，但是和别人有冲突又被同步掉了

**Comment by Heuer212** [comment]
同样的问题

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
